libraries/ms-common/test/test_x86_uart.py

```diff
@@ -0,0 +1,138 @@
+import os
```

Comment:
Please use python3.

Reply:
The script can now be run by the python3 interpreter. The only things that needed to be changed were the print statements and the thread module (it is now using _thread instead of thread).